src/org/omegat/gui/exttrans/MachineTranslateTextArea.java

```diff
@@ -147,18 +201,29 @@ private void startSearchThread(SourceTextEntry newEntry, boolean force) {
     @Override
     protected void setFoundResult(final SourceTextEntry se, final MachineTranslationInfo data) {
         UIThreadsUtil.mustBeSwingThread();
-
         if (data != null && data.result != null) {
-            displayed.add (data);
-            setText(getText() + data.result + "\n<" + data.translatorName + ">\n\n");
+            displayed.add(data);
+            displayed.sort(Comparator.comparing(info -> info.translatorName));
```

Comment:
Implementing Comparable on MachineTranslationInfo suggests you have to define comparability over all members, but that doesn't make a lot of sense (why should the translation itself be sorted lexicographically?).

Since we only care about sorting by translator name, I used this formulation and removed Comparable.